models/nc-news.articles.model.js

```diff
@@ -1,18 +1,13 @@
 const db = require('../db/connection');
 const format = require('pg-format');
 
-exports.selectTopics = async () => {
-  const sql = `SELECT * FROM topics`;
-
-  const topics = await db.query(sql);
-  return topics.rows;
-};
-
 exports.selectArticleById = async (articleId) => {
-  const sql = `SELECT name AS author, title, article_id, body, topic, created_at, votes
+  const sql = `SELECT name AS author, COUNT(comments.body) AS comment_count ,title, articles.article_id, articles.body, topic, articles.created_at, articles.votes
   FROM articles 
   JOIN users ON articles.author = users.username
-  WHERE article_id = $1;`;
+  JOIN comments ON comments.article_id = articles.article_id
+  WHERE articles.article_id = $1
+  GROUP BY articles.article_id, name, title, articles.article_id, articles.body, topic, articles.created_at, articles.votes;`;
```

Comment:
As you're grouping by the `articles` primary key you can skip the other `articles` columns here as the PK guarantees them to be unique. Up to you though, totally fine to be explicit 👍